themes/classic/_dev/package.json

```diff
@@ -33,4 +33,4 @@
         "webpack": "^2.2.1",
         "webpack-sources": "^0.1.0"
     }
-}
\ No newline at end of file
+}
```

Comment:
This is due to the newline at end of file adder of my IDE.

BTW, it's better to have this as default.